lib/rust/prelude/src/lib.rs

```diff
@@ -334,6 +335,13 @@ impl<T> RefCellOptionOps<T> for RefCell<Option<T>> {
     fn set(&self, val: T) {
         *self.borrow_mut() = Some(val);
     }
+
+    fn set_if_empty_or_warn(&self, val: T) {
+        if self.borrow().is_some() {
+            WARNING!("Trying to set value that was already set.")
```

Comment:
as this is a warning, not crash with a stack trace, I think it wold be beneficial to print these values in the warning.

Reply:
I don't think it's possible - closures does not implement `Debug`, so they can't be printed.

Reply:
I'm not talking about closures here, I'm talking about "values". I think my comment is still valid.

Reply:
Fixed